crates/katana-core/src/sequencer.rs

```diff
@@ -109,6 +111,20 @@ impl KatanaSequencer {
 
         Ok((tx_hash, contract_address))
     }
+
+    pub async fn starknet_get_storage_at(
+        &self,
+        contract_address: ContractAddress,
+        storage_key: StorageKey,
+    ) -> Result<FieldElement, FromByteSliceError> {
+        let storage = self
+            .state
+            .lock()
+            .unwrap()
+            .get_storage_at(contract_address, storage_key)
+            .unwrap();
+        FieldElement::from_byte_slice_be(storage.bytes())
+    }
```

Comment:
```suggestion
        self
            .state
            .lock()
            .unwrap()
            .get_storage_at(contract_address, storage_key)
    }
```

lets remove the marshaling logic from here and put it in the rpc. so methods here return starknet_api and/or blockifier types